src/avt/VisWindow/Colleagues/avtTimeSliderColleague.C

```diff
@@ -712,8 +713,46 @@ avtTimeSliderColleague::SetText(const char *formatString, const char *timeFormat
         strcpy(timeFormatString, timeFormat);
     }
 
+    //
+    // The two variables here are poorly named. One is the textFormatString
+    // which holds the text part of the string like "Time=" or "Cycle=", etc.
+    // The other is timeFormatString which holds a printf-style format
+    // string indicating how to format the numerical value associated with
+    // time.
+    //
+
+    //
+    // We have two possible places the printf-format string could be specified.
+    // One is part of the textFormatString itself (the new way) as in
+    // "Time=$time%6.4f" and the other (the old way) is in the separate
+    // timeFormatString. Which to use?
+    //
+
+    // 
+    // We first check for possible printf-style format string within the
+    // textFormatString itself. If we find something string there, we just
```

Comment:
find something string? Maybe typo? Or am I not reading the sentence right?

Reply:
Thanks fixed now.